src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -499,24 +591,18 @@ class FastSync(
         val now = Instant.now()
         val peers = unassignedPeers
           .filter(p => peerRequestsTime.get(p).forall(d => d.plusMillis(fastSyncThrottle.toMillis).isBefore(now)))
-        val blockChainPeers = blockChainOnlyPeers.toSet
-        (peers -- blockChainPeers)
-          .take(maxConcurrentRequests - assignedHandlers.size)
-          .toSeq.sortBy(_.ref.toString())
-          .foreach(assignWork)
         peers
```

Comment:
Minor detail: With this change the `blockChainOnlyPeers` was left unused, should we remove it?